safe/impact_function/impact_function.py

```diff
@@ -675,7 +756,7 @@ def callback(self):
 
             self.progress_callback(current, maximum, message=None)
 
-        :rtype: function
+        :rtype: function()
```

Comment:
hum, if I read this docstring, I might understand that the function is evaluated in the return.

and according to Python, it's a function:
```
def foo(): return '1'
type(foo)
<type 'function'>
type(1)
<type 'int'>
```

Reply:
I am not sure also for this. But PyCharm shows a warning if I only use `function` only. It says it's not callable. Somehow, I put the brackets, and the warnings are gone. What's the situtation in your PyCharm?

Reply:
Hum, I think it's still a function, but I guess PyCharm is confused because the same `callback` function has the `@property` decorator. And it's true a property can't be called. But in this case, the property is a function.

I understand what you mean, I got the same issue, but I let it as I didn't know how to fix it.
Do what you want ;-)
IMHO, we should fix this `property` which isn't a real property but a function maybe. But I'm lazzy to do that, very low priority.